taxcalc/functions.py

```diff
@@ -184,11 +185,18 @@ def CapGains(p23250, p22250, _sep, ALD_Investment_ec, ALD_StudentLoan_HC,
     c23650 = p23250 + p22250
     # limitation on capital losses
     c01000 = max((-3000. / _sep), c23650)
-    # compute ymod* variables
+    # compute ymod1 variable that is included in AGI
     ymod1 = (e00200 + e00700 + e00800 + e00900 + e01400 + e01700 +
              (1 - ALD_Investment_ec) * (e00300 + e00600 +
                                         c01000 + e01100 + e01200) +
              e02000 + e02100 + e02300)
+    if CG_nodiff != 0.:
```

Comment:
I understand the difficulties now. thanks @martinholmer. I say we merge this now and then think about relaxing the constraint later as necessary to other users.